client/state/sites/selectors/build-seo-title.js

```diff
@@ -27,15 +19,17 @@ export default ( titleFormats, type, { site, post = {}, tag = '', date = '' } )
 				buildTitle( 'posts', {
 					siteName: site.name,
 					tagline: site.description,
-					postTitle: get( post, 'title', '' ),
-				} ) || get( post, 'title', '' )
+					postTitle: post.title ?? '',
+				} ) ||
+				post.title ||
```

Comment:
It's odd that sometimes we use nullish coalescing and sometimes we use logical OR for the same thing 🤔

Reply:
We can't use `??` everywhere because `buildTitle` returns empty `''` string when it 'fails'. But we could replace the `post.title ?? ''` you're commenting on with `post.title || ''`. But it doesn't have a practical impact, it's a matter of aesthetics.